### PR TITLE
Document annotations to customize the endpoints shown in the Okteto UI

### PR DESCRIPTION
**Pull request by pchico83**
Document the annotation `dev.okteto.com/endpoints` coming in 1.7.0.
Also document the annotation `dev.okteto.com/hide-from-ui: true`, that it's already supported was it wasn't documented before.

### CODE REVIEW

File: src/content/cloud/ssl.mdx [hunk 1]
```diff
@@ -71,4 +71,16 @@ spec:
...
+## Customizing the endpoints shown in the Okteto UI
+
+Okteto provides additional annotations that enable you to customize the endpoints shown in the Okteto UI.
+
+To hide specific endpoints from the Okteto UI, you can use the `dev.okteto.com/hide-from-ui: true` annotation.
+Simply apply this annotation to your `Ingress` or `VirtualService` resource, and the endpoints associated with that resource will not be displayed in the Okteto UI.
+
+If you want to replace the default endpoints for an `Ingress` or `VirtualService` resource, you can use the `dev.okteto.com/endpoints` annotation.
+This annotation allows you to define a list of endpoints, separated by commas, that will replace the default endpoints.
+Additionally, the value of this annotation expands the `OKTETO_NAMESPACE` and `OKTETO_DOMAIN` environment variables, providing an easy way to make your endpoints portable between namespaces.
```
Comment (rberrelleza) [inline src/content/cloud/ssl.mdx:85]: Can we add example code for these?

Also, will this work for docker compose files? If yes, please include a sample too

File: src/content/self-hosted/install/releases.mdx [hunk 2]
```diff
@@ -5,6 +5,13 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.7.0
+TBD
+
+### Features
+
+- Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:13]: @omnomagonz what is the term for the UI? Okteto UI or Dashboard? I've seen both in the docs 😛

Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:13]: If we agree to that term sounds good. But then we'll need someone to go over the docs and make sure this is consistent 😀. Can you drive that?

Comment (omnomagonz) [reply src/content/self-hosted/install/releases.mdx:13]: Sure (what did I dooooooooooooo!?)

It likely aligns well with my existing effort to review docs so I'll make a note to check for this language 👍

Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:13]: yeah, I figured we can bundle this on that effort. It doesn't have to be as part of this PR

File: src/content/cloud/ssl.mdx (hunk 1 above)
Comment (pchico83) [reply src/content/cloud/ssl.mdx:88]: added sample

**Pull request merged**